### PR TITLE
Reduce default verbosity and replace bananalyzer's `-q` arg with `-v`

### PR DESCRIPTION
**Pull request by KhoomeiK**

### CODE REVIEW

File: bananalyzer/__main__.py [hunk 1]
```diff
@@ -129,10 +129,10 @@ def parse_args() -> Args:
         help="A list of ids to skip tests on, separated by commas",
     )
     parser.add_argument(
-        "-q",
-        "--quiet",
+        "-v",
+        "--verbose",
```
Comment (asim-shrestha) [inline bananalyzer/__main__.py:133]: can we also add `--v`? to match what pytest does

File: bananalyzer/runner/runner.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def run_tests(
         args = (
             test_file_names
             + ["-s"] * pytest_args.s
-            + (["-q"] if pytest_args.q else ["-vvv"])
+            + (["-vvv"] if pytest_args.v else ["-v"])
```
Comment (asim-shrestha) [inline bananalyzer/runner/runner.py:133]: -v is enough to not show multiple error lines?

Comment (KhoomeiK) [reply bananalyzer/runner/runner.py:133]: yup, tested

File: bananalyzer/__main__.py (hunk 1 above)
Comment (KhoomeiK) [reply bananalyzer/__main__.py:133]: nah

**Pull request merged**